.github/actions/create-release-candidate-v1/action.yml

```diff
@@ -18,21 +18,73 @@ inputs:
     description: 'Whether or not the version should be locked to axe-core'
     required: false
     default: 'false'
+  should-checkout:
+    description: 'Whether or not the action should checkout the repository'
+    required: false
+    default: 'true'
   docs-repo:
     description: 'The name of the repo where the release notes live'
     required: false
     default: 'null'
+  docs-issue-assignees:
+    description: 'Assignees for the release notes issue'
+    required: false
   docs-labels:
     description: 'Labels for release notes issue. Comma-delimited list of labels (e.x. "release,PRIORITY: high")'
     required: false
-  should-checkout:
-    description: 'Whether or not the action should checkout the repository'
+  docs-issue-project-number:
+    description: 'Project number for the release notes issue'
+    required: true
+  docs-issue-column-name:
+    description: 'Column name for the release notes issue (e.g. "Todo")'
+    required: true
+  release-issue-assignees:
+    description: 'Assignees for the release issue'
     required: false
-    default: 'true'
+  release-issue-labels:
+    description: 'Labels for the release issue. Comma-delimited list of labels (e.x. "release,PRIORITY: high")'
+    required: false
+    default: 'PRIORITY: high,release'
+  release-issue-project-number:
+    description: 'Project number for the release issue'
+    required: true
+  release-issue-column-name:
+    description: 'Column name for the release issue (e.g. "New")'
+    required: true
 
 runs:
   using: 'composite'
   steps:
+    - name: Validate required inputs
```

Comment:
Why is this even needed? Required inputs are already validated before execution happens on Github.